### PR TITLE
:sparkle: add a new option to use spdx identifier in reports

### PR DESCRIPTION
**Pull request by etiennecadicidean**
- SPDX identifier is essential to make license match between CI tools.
 - Using the proper --use_spdx_licenses, the user can get those licenses
in the proper format to use reports in tools such as Sonar License Check plugins

### CODE REVIEW

**Comment by etiennecadicidean** [comment]
Hello there @xtreme-shane-lattanzio 
the CI fails tests while everything is fine in the docker image using : 

```
./dlf rake spec
./dlf bundle exec rake features

```
Can you figure out what it takes to make the PR approved ?

File: lib/license_finder/license/definitions.rb [hunk 1]
```diff
@@ -46,6 +46,7 @@ def apache1_1
         License.new(
           short_name: 'Apache1_1',
           pretty_name: 'Apache 1.1',
+          spdx_id: 'Apache-1.0',
```
Comment (xtreme-debbie-chen) [inline lib/license_finder/license/definitions.rb:49]: Is this correct? This is the license for Apache 1.1. I would expect the spdx id to be  `Apache-1.1` (from the [spdx license list](https://spdx.org/licenses/))

Comment (etiennecadicidean) [reply lib/license_finder/license/definitions.rb]: totally right, I just fixed it. Thank you for reviewing !

Comment (etiennecadicidean) [reply lib/license_finder/license/definitions.rb]: i doubled check the other ones, should be okay.

**Comment by xtreme-shane-lattanzio** [comment]
@etiennecadicidean Was just waiting for CI and now merged! Thanks for another one!!

**Pull request merged**